pages/index.tsx

```diff
@@ -1,12 +1,12 @@
-import Layout from "../components/Layout";
-import Navbar from "../components/Navbar";
+import Layout from '../components/Layout'
+import Navbar from '../components/Navbar'
 
 // modules
-import HeroSection from "../modules/HomePage/hero";
-import MakeGoodThings from "../modules/HomePage/makeGoodThings";
-import WhatWeDo from "../modules/HomePage/WhatWeDo";
-import OurPartners from "../modules/HomePage/ourPartners";
-import Footer from "../components/Footer";
+import HeroSection from '../modules/HomePage/HeroSection'
+import MakeGoodThings from '../modules/HomePage/MakeGoodThings'
+import WhatWeDo from '../modules/HomePage/WhatWeDo'
+import OurPartners from '../modules/HomePage/OurPartners'
+import Footer from '../components/Footer'
```

Comment:
@sithumonline @LayanJay can you guys stick to one comma pattern

Reply:
in prettierrc files, it defined as single-quote true

Reply:
Ohh! okay then